.github/workflows/ci.yml

```diff
@@ -23,6 +23,15 @@ jobs:
       with:
         submodules: true
 
+    - name: Changelog check
```

Comment:
I think we should make a separate job for this step, so that the feedback is more clearly exposed on the PR page and has a better separation of duties considering it has nothing to do with `Build and Test`